static/src/js/providers/TourContextProvider/TourContextProvider.jsx

```diff
@@ -11,13 +11,11 @@ const { disableSiteTour } = getApplicationConfig()
 const isSiteTourEnabled = disableSiteTour === 'false'
 
 const TourContextProvider = ({ children }) => {
-  const [runTour, setRunTour] = useState(false)
+  const [runTour, setRunTour] = useState(0)
   useEffect(() => {
     const isLocalhost = window.location.hostname === 'localhost'
```

Comment:
Might be nice if we pulled some of these into `static/src/js/util/url/url.js` since they're just one liners and pretty specific to what we have to let Joyride know perhaps its not worth it